tests/org.corpus_tools.hexatomic.it.tests/src/main/java/org/corpus_tools/hexatomic/it/tests/TestGraphEditor.java

```diff
@@ -170,16 +243,91 @@ public String getFailureMessage() {
 
   @Test
   @Order(1)
-  void testShowSaltExample() {
+  void testShowSaltExample()
+      throws NoSuchMethodException, IllegalAccessException, InvocationTargetException {
 
+    // Open example and maximize part
     openDefaultExample();
 
-    Graph g = bot.widget(widgetOfType(Graph.class));
-    assertNotNull(g);
+    SWTBotView view = bot.partByTitle("doc1 (Graph Editor)");
+    view.maximise();
+    bot.waitUntil(new PartMaximizedCondition(view.getPart()));
 
     // Check all nodes and edges have been created
-    assertEquals(23, g.getNodes().size());
-    assertEquals(22, g.getConnections().size());
+    bot.waitUntil(new NumberOfNodesCondition(23));
+    bot.waitUntil(new NumberOfConnectionsCondition(22));
+    
+    Graph g = bot.widget(widgetOfType(Graph.class));
+    assertNotNull(g);
+    SWTUtils.display().syncExec(g::forceFocus);
+
+    final Viewport viewPort = (Viewport) SWTUtils.invokeMethod(g, GET_VIEWPORT);
+    
+    Point origLocation = (Point) SWTUtils.invokeMethod(viewPort, GET_VIEW_LOCATION);
+
+    // Initially, the zoom is adjusted to match the height, so moving up/down should not do anything
+    keyboard.pressShortcut(Keystrokes.DOWN);
+    bot.waitUntil(
+        new ViewLocationReachedCondition(viewPort, new Point(origLocation.x, origLocation.y)));
+    keyboard.pressShortcut(Keystrokes.UP);
+    bot.waitUntil(
+        new ViewLocationReachedCondition(viewPort, new Point(origLocation.x, origLocation.y)));
+
+    // Zoom in so we can move the view with the cursors
```

Comment:
```suggestion
    // Zoom in so we can move the view with the arrow keys
```

Sorry, slightly pedantic.